packages/cli/src/commands/inspect.ts

```diff
@@ -34,6 +37,10 @@ const help = () => {
     -d, --debug                    Debug mode [off]
     --no-color                     No color mode [off]
     -S, --scope                    Set a custom scope
+    --timeout=${chalk.bold.underline(
+      'TIME'
+    )}                 Time to wait for deployment completion [3m]
+    --wait                         Blocks until site has deployed
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** reword‏

```suggestion
    --wait                         Blocks until deployment completes
```

I think the wording here should more clearly allow for deployments that suceed or fail.

Reply:
Fixed!